src/test/scala/cromwell/engine/db/slick/SlickDataAccessSpec.scala

```diff
@@ -262,9 +262,12 @@ class SlickDataAccessSpec extends FlatSpec with Matchers with ScalaFutures {
             new Call(callAlias, "call.name", task, Set.empty[FullyQualifiedName], Map.empty, None)
           }
 
+        val callKey = ExecutionDatabaseKey(call.fullyQualifiedName, None)
+        val shardKey = ExecutionDatabaseKey(call.fullyQualifiedName, Option(0))
```

Comment:
Disagree on `None`. Agree on the `Option(0)` but then worry it'd be weirdly inconsistent.